lwt/cohttp_lwt.ml

```diff
@@ -298,7 +298,14 @@ module Make_server(IO:IO)
     Filename.concat docroot rel_path
 
   let respond ?headers ?(flush=true) ~status ~body () =
-    let encoding = Cohttp_lwt_body.transfer_encoding body in
```

Comment:
I wouldn't mind seeing this in its own function for testing purposes. Although I wouldn't want the user to be exposed to it...